pom.xml

```diff
@@ -37,7 +37,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <awssdk.version>2.13.18</awssdk.version>
+        <awssdk.version>2.15.0</awssdk.version>
```

Comment:
regularly bumping sdk allows handlers be up to date with the latest in the base sdk. can also do a range of versions but then builds arent really reproducible. doing a bit of research into http://www.mojohaus.org/versions-maven-plugin/ now to see if this can make this less of a manual thing

Reply:
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/267#issuecomment-632796555

Reply:
Version ranges in Bill of Material (BOM) poms aren't supported: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms

One has to specify a specific version number for the project imported.

Reply:
We should probably specify a  bill of materials in the pom.xml for the resource provider projects as well. Maven's dependency management works on a "nearest wins" strategy: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html, so if we had explicitly declared dependencies for the entire aws sdk package in the resource provider as well, that version would override since it is closer to the root.